### ISSUE TITLE
No "exports" main defined in package.json

### ISSUE THREAD

**Issue by gusfune** [devrnt/react-use-intercom#124]
**Describe the bug**
This bug happens when running on NextJS 10.0.0 when building a project on node v14.14.0.
It throws the error:

```
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: No "exports" main defined in /project/node_modules/react-use-intercom/package.json
    at throwExportsNotFound (internal/modules/esm/resolve.js:290:9)
    at packageExportsResolve (internal/modules/esm/resolve.js:479:7)
    at resolveExports (internal/modules/cjs/loader.js:432:36)
    at Function.Module._findPath (internal/modules/cjs/loader.js:472:31)
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:867:27)
    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.J92l (/project/.next/server/pages/[REDACTED])
    at __webpack_require__ (/Users/gus/Developer/storefront/.next/server/pages/[REDACTED]) {
  type: 'NodeError',
  code: 'ERR_PACKAGE_PATH_NOT_EXPORTED'
}
```

**To Reproduce**
1. Update the package to 1.1.3.
2. Build the project.
3. See error

**Expected behavior**
The project builds as before the update.

**Desktop (please complete the following information):**
 - OS: macOS 10.15.7
 - Browser: SSR
 - Version 1.1.3

**Additional context**
Project builds if editing on `node_modules` the project directly by removing from `package.json`:
```
"exports": {
    ".": {
      "import": "./dist/react-use-intercom.esm.js"
    }
  },
```
This is basically reverting the commit 984288bfbbba0d27f2a596568213593c75a3a42d.

**Comment by devrnt** [comment]
Thanks for the detailed issue report, really appreciate it. Main issue here is that `next build` needs a `require` entry since it is a node environment. 

I'll fix this right away.

**Comment by gusfune** [comment]
That was quick, thanks a lot! 👍🏽

**Comment by devrnt** [comment]
Published [v1.1.4](https://github.com/devrnt/react-use-intercom/releases/tag/v1.1.4)
